src/containers/Lobby.js

```diff
@@ -37,7 +42,19 @@ class Lobby extends PureComponent {
                 <Paper
                   zDepth={1}
                   style={{ padding: '12px 24px' }}>
-                  <h4>{ game.title }</h4>
+                    <h4>{ game.title }</h4>
+                    <div>
+                      { game.playerIds.length < 2 &&
```

Comment:
Just a question. `game.playerIds.length < 2 &&` what? what else are you comparing against?